template.json

```diff
@@ -1039,13 +1039,19 @@
             }
         },
         "vehicle": {
-            "templates": ["common", "skills", "attributes", "limits", "armor", "matrix", "movement", "physical_combat_values"],
+            "templates": [ "common", "skills", "attributes", "limits", "armor", "matrix", "movement", "physical_combat_values" ],
             "vehicleType": "ground",
             "controlMode": "autopilot",
             "environment": "speed",
             "isDrone": false,
             "isOffRoad": false,
             "driver": "",
+            "modCategorySlots_Body": 0,
```

Comment:
Does this need to be a bunch of top level actor properties?

To improve organisational structure I suggest moving these into an object structure
```
modCategorySlots: {
 body: 0,
 ...
}
```

Reply:
Thanks for the feedback. Resolved with latest commit.